### PR TITLE
Edit main, settings pages

### PR DESCRIPTION
**Pull request by NeBioNik**
Enable EN search
Change RU tittle page name
Edit main page
Edit some settings page
Add more explanation to topic control
Edit avaliable proxy

### CODE REVIEW

File: content/configuration/proxy-server.md [hunk 1]
```diff
@@ -3,17 +3,21 @@ title: "Прокси-сервер"
...
 
-Прокси-сервер, например, можно использовать, если ваш провайдер блокирует доступ к ресурсам rutracker.org.
+Прокси-сервер можно использовать, если ваш провайдер блокирует доступ к ресурсам rutracker.org.
 
 > **ВНИМАНИЕ!** Прокси-сервер **НЕ** нужно включать без надобности! Лучше подберите рабочие адреса для форума и API (см. [Авторизация на форуме]({{< relref "/configuration/authorization-forum" >}})).  
 
 Можно задействовать использование прокси отдельно для форума и API, для этого установите соответствующие флажки. Заполните поля «IP-адрес» и «Порт». Если требуется авторизация на прокси, укажите эти данные в полях «Логин» и «Пароль». В этом случае **только** web-TLO будет работать через заданный в настройках прокси-сервер.  
 
-> **ВНИМАНИЕ!** Предпочтительным вариантом для обхода блокировки является использование SOCKS5-прокси. Вводим тип прокси-сервера — «SOCKS5H», адрес — «px1.blockme.site», порт — «33128».
+> **ВНИМАНИЕ!** Предпочтительным вариантом для обхода блокировки является использование SOCKS5(H)-прокси.
```
Comment (Orhideous) [inline content/configuration/proxy-server.md]: Only SOCKS5H

File: content/configuration/proxy-server.md [hunk 2]
```diff
@@ -3,17 +3,21 @@ title: "Прокси-сервер"
...
 
-> **ВНИМАНИЕ!** Предпочтительным вариантом для обхода блокировки является использование SOCKS5-прокси. Вводим тип прокси-сервера — «SOCKS5H», адрес — «px1.blockme.site», порт — «33128».
+> **ВНИМАНИЕ!** Предпочтительным вариантом для обхода блокировки является использование SOCKS5(H)-прокси. 
 
-> **ВНИМАНИЕ!** Не рекомендуется использовать прокси-сервер при обращении к API, так как это существенно замедляет получение сведений о раздачах.
+Рекомендуемые прокси от хранителей:
+* `gateway.keepers.tech:60789` (SOCKS5H)
+* `gateway.keeps.cyou:2081` (SOCKS5H)
+
+> **ВНИМАНИЕ!** Крайне не рекомендуется использовать прокси-сервер при обращении к API, так как это существенно замедляет получение сведений о раздачах.
```
Comment (Orhideous) [inline content/configuration/proxy-server.md]: It's outdated advice. Let's drop it.

**Review by Orhideous — CHANGES_REQUESTED** [review]
All fine, except two minor things

File: content/configuration/proxy-server.md (hunk 1 above)
Comment (NeBioNik) [reply content/configuration/proxy-server.md]: Fixed, if I understood correctly.

**Review by Orhideous — APPROVED** [review]

**Pull request merged**